data/templates/set_bmc_credentials.sh

```diff
@@ -1,9 +1,23 @@
 #!/usr/bin/env bash
 
+channel=''
+function set_channel()
+{
+    for i in {1..15}; do
+        ipmitool -I lanplus -H 172.31.128.2 -U admin -P admin user list $i &>/dev/null
+        status=$?
+        if [ "$status" -eq "0" ] ; then
+            channel=$i
+            break
```

Comment:
It's doubtful we'll hit this scenario, but it seems like this won't handle the case where $status is non-zero for all channel's 1-15?